hwpf/fapi2/src/plat/plat_error.C

```diff
@@ -171,7 +171,13 @@ void process_HW_callout(FFDC &ffdc, const bool deconfRefTarget)
 		// BMC context.
 		if ((hwcallout.hwid == "PROC_REF_CLOCK") ||
 		    (hwcallout.hwid == "PCI_REF_CLOCK")) {
-			process_clock_callout(ffdc, hwcallout, deconfRefTarget);
+			if (clock_redundancy_is_enabled()) {
+				clock_callout_info(ffdc, hwcallout);
```

Comment:
this is also part of processing clock callout, right?

why cant we do this inside process_clock_callout()

Reply:
This will get high level view on callout flow from this spercial callout handling function.

Reply:
> this is also part of processing clock callout, right?
> 
> why cant we do this inside process_clock_callout()

Done